odxtools/writepdxfile.py

```diff
@@ -225,6 +225,21 @@ def write_pdx_file(
 
             del jinja_vars["flash"]
 
+        # write the ECU-config objects
+        ecu_config_tpl = jinja_env.get_template("ecu_config.odx-e.xml.jinja2")
+        for ecu_config in database.ecu_configs:
+            zf_file_name = f"{ecu_config.short_name}.odx-e"
+            zf_file_cdate = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
```

Comment:
What is this?

Reply:
this is required for the `index.xml` file of the PDX. it might be better to track the creation date with each category object of  the database, but since this information is IMO quite irrelevant (`.admin_data.doc_revisions` is a much better way to track version information), odxtools uses the time the PDX file is written as the timestamp for `index.xml`...